schema/src/BT01/BT01CreditClass.yaml

```diff
@@ -0,0 +1,155 @@
+id: https://framework.regen.network/schema/BTCreditClass
+name: BTCreditClass
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+imports:
+  - linkml:types
+default_curi_maps:
+  - semweb_context
+default_prefix: rfs
+default_range: string
```

Comment:
If the default range is `string` I don't think we need to specify `range: string` below on individual slots